content/api/2-client.mdx

```diff
@@ -14,7 +14,7 @@ config = {
   user?: string, // default process.env.PGUSER || process.env.USER
   password?: string or function, //default process.env.PGPASSWORD
   host?: string, // default process.env.PGHOST
-  database?: string, // default process.env.PGDATABASE || process.env.USER
+  database?: string, // default process.env.PGDATABASE || process.env.PGUSER || process.env.USER
```

Comment:
Since brianc/node-postgres#1679, I think this should be `process.env.PGDATABASE || user`.

```suggestion
  database?: string, // default process.env.PGDATABASE || user
```